android/app-newm/src/main/java/io/newm/screens/home/MusicCarousel.kt

```diff
@@ -0,0 +1,112 @@
+package io.newm.screens.home
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyRow
+import androidx.compose.foundation.lazy.items
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import coil.compose.AsyncImage
+import io.newm.core.resources.R
+import io.newm.core.theme.*
+
+data class MusicModel(
+    val title: String,
+    val subtitle: String,
+    val imageUrl: String,
+    val imageText: String? = null
+)
+
+@Composable
+fun MusicCarousel(
+    title: String,
+    musicModels: List<MusicModel>,
+    onViewDetails: (MusicModel) -> Unit,
+    onViewAll: (() -> Unit)? = null
+) {
+    Column(
+        modifier = Modifier
+            .padding(top = 28.dp)
+            .fillMaxWidth()
+    ) {
+        Row(modifier = Modifier.padding(start = 12.dp, end = 12.dp)) {
```

Comment:
For places where you need to add padding to the start and end, you could use something like horizontal / vertical
`Modifier.padding(horizontal = 12.dp)`

Reply:
Done!